src/qt/multisigdialog.cpp

```diff
@@ -34,9 +34,11 @@
 #include <QArgument>
 #include <QtGlobal>
 #include <QString>
+#include <multisignature.h>
```

Comment:
improper include ordering/styling, this should be move up to be with the other in-tree includes.